packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -4,13 +4,13 @@
 
 #### docs
 
-- Docs/Gemfile.lock to reduce vulnerabilities ([PR #4147](https://github.com/forcedotcom/salesforcedx-vscode/pull/4147))
+- We made some updates under the hood. ([PR #4147](https://github.com/forcedotcom/salesforcedx-vscode/pull/4147))
 
 #### salesforcedx-vscode-core
 
-- Add additional context for scope of generate manifest command ([PR #4208](https://github.com/forcedotcom/salesforcedx-vscode/pull/4208))
+- We fixed an issue where you could run the `SFDX Generate Manifest File` command to create a manifest file from any folder in your project. The command is now only available by right clicking on the `force-app`, package directory folder.  ([PR #4208](https://github.com/forcedotcom/salesforcedx-vscode/pull/4208))
```

Comment:
nit: I think it might be more accurate to say the command is available for any folder or file under the `force-app` folder rather than for just the `force-app` package directory folder.